### ISSUE TITLE
Add RPMs and DEBs to releases

### ISSUE THREAD

**Issue by JonZeolla** [SeisoLLC/zeek-kafka#3]
We should be able to automatically include artifacts such as RPMs and DEBs to our releases, using tags from the versions specified in the package.

**Comment by JonZeolla** [comment]
Depends on #10

**Comment by ottobackwards** [comment]
I've done RPM installs of plugins, and we do zkg now.  I don't know how I would mix the two.  I wonder if there are any other folks who do?

**Comment by JonZeolla** [comment]
My thought is to appeal to environments that prefer to host packages and install using builtin tools.  We could also add a `zkg` bundle to the releases?

**Comment by JonZeolla** [comment]
I know that there have been projects which used `metron-bro-plugin-kafka` which made their own RPMs/DEBs.

**Comment by ottobackwards** [comment]
I know how to do it, but not in a zkg environment.  IE>  what would be the best practice?  just deploy it?  try running zkg after unpacking it?  we should as on #development

**Comment by JonZeolla** [comment]
My plan was that the rpms and debs would provide an install method independent of zsh

**Comment by ottobackwards** [comment]
Ok, can you put that explicitly in the issue description?

**Comment by JonZeolla** [comment]
Done

**Comment by ottobackwards** [comment]
Maybe we should add "install" images for supported systems to our set.  So that we can run quick tests to be sure the debs install in ubuntu? and centos?

**Comment by JonZeolla** [comment]
I agree that should be a part of this work. Since there aren't really hierarchies with GitHub issues what do you feel if we just make a list of issues and indicate which are blocked with a label, and be specific about the prereq issues in the first comment?

**Comment by ottobackwards** [comment]
sure

**Comment by ottobackwards** [comment]
that or a feature branch

**Comment by JonZeolla** [comment]
For a project this small I have a hard time with using longer lived feature branches, but if the scenario arises I'm fine with that too
